CMakeLists.txt

```diff
@@ -24,11 +24,21 @@ include(Packages)
 
 # This has to be defined after Packages is included, because it relies on the
 # discovered path to the python executable.
-set(PXR_PYTHON_SHEBANG "${PYTHON_EXECUTABLE}" 
-    CACHE 
-    STRING
-    "Replacement path for Python #! line."
-)
+if (NOT PXR_PYTHON_SHEBANG)
+    set(PXR_PYTHON_SHEBANG "${PYTHON_EXECUTABLE}"
+        CACHE
+        STRING
+        "Replacement path for Python #! line."
+    )
+endif()
+
+if (NOT PYTHON_COMMAND_NAME)
+    set(PYTHON_COMMAND_NAME "${PYTHON_EXECUTABLE}"
```

Comment:
Ah. Slight nuance/pickup by @seando-adsk - this actually _isn't_ the existing behaviour. Existing behaviour would be to use the literal string 'python'

TL;DR unexpected/unintended behaviour change. I'll add another commit to this branch to change that to better match 1:1

Reply:
(aaaaand done)